utils/perl5lib/Streams/TemplateGeneric.pm

```diff
@@ -463,8 +465,8 @@ sub GetDataFilepath {
   if ( ! defined($self->{'template'}) ) {
      die "${nm}:: a template has NOT been read in yet -- abort.\n";
   }
-  my $defaults_ref = $self->{'defaults'};
```

Comment:
I couldn't help myself and tidied up this file in a couple of places removing unused variables.